src/peergos/server/storage/IPFS.java

```diff
@@ -505,12 +505,17 @@ private Object retrieveAndParse(String path) throws IOException {
 
     private static byte[] get(URL target) throws IOException {
         HttpURLConnection conn = (HttpURLConnection) target.openConnection();
-        conn.setRequestMethod("GET");
+        conn.setDoOutput(true);
+        conn.setRequestMethod("POST");
```

Comment:
probably best to add a comment and link to ipfs commit on why POST is used

Reply:
Done